app/src/androidTest/java/com/github/se/signify/ui/screens/ChallengeHistoryScreenTest.kt

```diff
@@ -0,0 +1,63 @@
+package com.github.se.signify.ui.screens
+
+import androidx.compose.ui.test.assertIsDisplayed
+import androidx.compose.ui.test.junit4.createComposeRule
+import androidx.compose.ui.test.onNodeWithTag
+import androidx.compose.ui.test.performClick
+import com.github.se.signify.ui.navigation.NavigationActions
+import kotlinx.coroutines.runBlocking
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.mockito.Mockito.mock
+import org.mockito.Mockito.verify
+
+class ChallengeHistoryScreenTest {
```

Comment:
Very good test Classe : Testing the display of all the elements in the screen and the behaviour of the back arrow